internal/runner/command.go

```diff
@@ -0,0 +1,84 @@
+package runner
+
+import (
+	"os"
+	"os/exec"
+	"os/signal"
+	"slices"
+	"syscall"
+
+	"github.com/kballard/go-shellquote"
+)
+
+// runAndForwardSignal runs the command and forwards any signals received to the command.
+func runAndForwardSignal(cmd *exec.Cmd) error {
```

Comment:
@DrJosh9000 do you think it is a sensible approach to extract this bit from `main` function into a separate function?

Reply:
Yep, I think it makes sense as a function.